bundles/org.openhab.binding.guntamatic/README.md

```diff
@@ -33,165 +33,167 @@ Guntamatic Heating Systems supported as Thing Types:
-| `controlExtraWwHeat1`     | Trigger Extra Warm Water Circle 1 (`RECHARGE`)                                  | `String` |      |        🔐 W1         |   R/W    |   true   |
-| `controlExtraWwHeat2`     | Trigger Extra Warm Water Circle 2 (`RECHARGE`)                                  | `String` |      |        🔐 W1         |   R/W    |   true   |
-
-- <b id="f1">1)</b> ... `MANUAL` is supported by Biostar, Powerchip, Powercorn, Biocom, Pro as well as Therm only [↩](#a1)
+| Channel               | Description                                                                     | Type     | Unit | Security Access Level | ReadOnly | Advanced |
+| --------------------- | ------------------------------------------------------------------------------- | -------- | :--: | :-------------------: | :------: | :------: |
+| `boiler-approval`     | Set Boiler Approval (`AUTO`, `OFF`, `ON`)<sup id="a1">[1](#f1)</sup>)           | `String` |      |        🔐 W1         |   R/W    |   true   |
+| `program`             | Set Program (`OFF`, `NORMAL`, `WARMWATER`, `MANUAL`<sup id="a2">[2](#f2)</sup>) | `String` |      |        🔐 W1         |   R/W    |  false   |
+| `heat-circ-program-0` | Set Heat Circle 0 Program (`OFF`, `NORMAL`, `HEAT`, `LOWER`)                    | `String` |      |        🔐 W1         |   R/W    |   true   |
+| `heat-circ-program-1` | Set Heat Circle 1 Program (`OFF`, `NORMAL`, `HEAT`, `LOWER`)                    | `String` |      |        🔐 W1         |   R/W    |   true   |
+| `heat-circ-program-2` | Set Heat Circle 2 Program (`OFF`, `NORMAL`, `HEAT`, `LOWER`)                    | `String` |      |        🔐 W1         |   R/W    |   true   |
+| `heat-circ-program-3` | Set Heat Circle 3 Program (`OFF`, `NORMAL`, `HEAT`, `LOWER`)                    | `String` |      |        🔐 W1         |   R/W    |   true   |
+| `heat-circ-program-4` | Set Heat Circle 4 Program (`OFF`, `NORMAL`, `HEAT`, `LOWER`)                    | `String` |      |        🔐 W1         |   R/W    |   true   |
+| `heat-circ-program-5` | Set Heat Circle 5 Program (`OFF`, `NORMAL`, `HEAT`, `LOWER`)                    | `String` |      |        🔐 W1         |   R/W    |   true   |
+| `heat-circ-program-6` | Set Heat Circle 6 Program (`OFF`, `NORMAL`, `HEAT`, `LOWER`)                    | `String` |      |        🔐 W1         |   R/W    |   true   |
+| `heat-circ-program-7` | Set Heat Circle 7 Program (`OFF`, `NORMAL`, `HEAT`, `LOWER`)                    | `String` |      |        🔐 W1         |   R/W    |   true   |
+| `heat-circ-program-8` | Set Heat Circle 8 Program (`OFF`, `NORMAL`, `HEAT`, `LOWER`)                    | `String` |      |        🔐 W1         |   R/W    |   true   |
+| `ww-heat-0`           | Trigger Warm Water Circle 0 (`RECHARGE`)                                        | `String` |      |        🔐 W1         |   R/W    |   true   |
```

Comment:
There is a more common used abbreviation for ww (warm water) and that  is dhw  (domestic hot water).

Reply:
I'm not creating those abbreviations. They come directly like that from the heating systems. As different heating systems provide different data (there is no specification) I'm only transforming them 1:1 into channels (w/o adding any extra magic). Some data even change on the language that is set on the heating system.
![image](https://github.com/user-attachments/assets/f05b6634-2a51-4481-8030-7ac9b3978708)

Reply:
I thought so :-/ we have to deal with it, thanks for coming back to this!